### PR TITLE
change the spark version of dockerfile

### PR DESCRIPTION
**Pull request by sven0726**
### What is this PR for?
A few sentences describing the overall goals of the pull request's commits.
First time? Check out the contributing guide - https://zeppelin.apache.org/contribution/contributions.html

### What type of PR is it?
[Bug Fix | Improvement | Feature | Documentation | Hot Fix | Refactoring]

### Todos
* [ ] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/ZEPPELIN/
* Put link here, and add [ZEPPELIN-*Jira number*] in PR title, eg. [ZEPPELIN-533]

### How should this be tested?
Outline the steps to test the PR here.

### Questions:
* Does the licenses files need update?
* Is there breaking changes for older versions?
* Does this needs documentation?

### CODE REVIEW

File: scripts/docker/spark-cluster-managers/spark_mesos/Dockerfile [hunk 1]
```diff
@@ -14,8 +14,8 @@
 # limitations under the License.
 FROM centos:centos6
 
-ENV SPARK_PROFILE 2.0
-ENV SPARK_VERSION 2.0.0
+ENV SPARK_PROFILE 2.1
+ENV SPARK_VERSION 2.1.1
 ENV HADOOP_PROFILE 2.3
 ENV HADOOP_VERSION 2.3.0
```
Comment (1ambda) [inline scripts/docker/spark-cluster-managers/spark_mesos/Dockerfile]: should be 2.7 as well?

**Comment by 1ambda** [comment]
Hi @sven0726, Thanks for the contribution. I added a comment. 

- This pr is not related with CI, but if you want to modify code you need to setup travis CI for your zeppelin fork.

**Comment by 1ambda** [comment]
Hi @sven0726! Any update? :)

**Comment by felixcheung** [comment]
as in here: http://zeppelin.apache.org/contribution/contributions.html#continuous-integration

**Comment by sven0726** [comment]
@1ambda  Could you please check it again ?    

@felixcheung  Thank you for your guidance！

**Comment by 1ambda** [comment]
LGTM!

**Comment by 1ambda** [comment]
LGTM.

**Comment by Leemoonsoo** [comment]
LGTM and merge to master if no further comment.

**Comment by sven0726** [comment]
I am ok

**Pull request closed without merging**